studio/Dockerfile

```diff
@@ -7,19 +7,19 @@
 #    docker builder prune
 
 
-FROM node:14 as base
+FROM node:14-alpine as base
 WORKDIR /usr/src/app
 # Do `npm ci` separately so we can cache `node_modules`
 # https://nodejs.org/en/docs/guides/nodejs-docker-webapp/
 COPY package*.json ./
-RUN npm clean-install
+RUN npm clean-install && npm cache clean --force
 COPY . .
 
 FROM base as dev
 EXPOSE 8082
 CMD ["npm", "run", "dev"]
 
 FROM base as production
-RUN npm run build
+RUN npm run build && rm -rf .next/cache/webpack && npm prune --production
```

Comment:
I removed the internal Webpack cache here since this took up an additional 100MB and it wouldn't be reused in any form in this image.